zeebe-test-runner/src/test/kotlin/io/zeebe/bpmnspec/runner/zeebe/ZeebeTestRunnerTest.kt

```diff
@@ -39,7 +39,7 @@ class ZeebeTestRunnerTest {
     }
 
     @Test
-    fun `Runner with ZeebeTestRunner should run the spec`() {
+    fun `Runner with ZeebeTestRunner should run the YAML spec`() {
```

Comment:
(optional) We could write these test cases as parameterized tests to avoid duplication. See here: https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-MethodSource

Reply:
I will not implement this suggestion right now.